manifests/sysbench/bench.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: sysbench-bench
+  labels: 
+    app: sysbench-bench
+spec:
+  parallelism: 3
+  template:
+    metadata:
+      labels:
+        app: sysbench-bench
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: sysbench
+        image: uhub.ucloud.cn/pingcap/sysbench:latest
```

Comment:
Can this image be customized?

Reply:
yes, add a section "More customizations" for this